templates/geosafe/analysis/create.html

```diff
@@ -1091,7 +1124,9 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
                     update_impact_layer(impact_id, false);
                     check_layer_empty();
                     {% if not user.username %}
-                        show_info_text('<span class="text-danger">You have not logged in yet</span>', '<span class="alert-danger">Please login to save analysis</span>', 'alert-danger');
+                        not_logged_warning = " <span class='alert-danger'> - Please <a href='#' data-toggle='modal' data-target='#SigninModal' role='button' >{% trans "sign in" %} </a> to save analysis </span>";
+                    {% else %}
```

Comment:
move these diffs to appropriate function in templates/geosafe/analysis/dynamic_scripts/side_panel.html